bazel/protobuf.bzl

```diff
@@ -89,24 +89,41 @@ def get_include_directory(source_file):
     else:
         return source_file.root.path if source_file.root.path else "."
 
-def get_plugin_args(plugin, flags, dir_out, generate_mocks):
+def get_plugin_args(
+        plugin,
+        flags,
+        dir_out,
+        generate_mocks,
+        plugin_name = "PLUGIN"):
```

Comment:
Nit: I'm wondering if it might make more sense to make `plugin_name` a positional argument and make the invocation pass "PLUGIN" as a value.

Reply:
Leaving it as is as we discussed in a chat.